src/components/customMdx/prism/prism-prisma.js

```diff
@@ -19,3 +19,5 @@ Prism.languages.insertBefore('prisma', 'function', {
 Prism.languages.insertBefore('prisma', 'punctuation', {
   'type-args': /\b(?:references|fields):/,
 })
+
+Prism.languages.json5 = Prism.languages.js
```

Comment:
What does this do and why?

Reply:
```json5
[{ email: 'yewande@prisma.io' }, { email: `raheem@gmail.com` }]
```

Reply:
Ohh interesting - so `json5` is a workaround for stupid prettier rules basically? We should document that somewhere in the guide pages then.